src/wdio/services/AxeService.js

```diff
@@ -9,20 +9,24 @@ let axeCoreSrc;
 */
 export default class AxeService {
   // eslint-disable-next-line class-methods-use-this
-  before(config) {
-    const axeConfig = {
-      inject: true, // True if axeCore script should be injected on each test page.
-      ...(config.axe || {}),
-    };
-
+  before() {
     browser.addCommand('axe', (options = {}) => {
       // Conditionally inject axe. This allows consumers to inject it themselves
       // in the test examples which would slightly speed up test runs.
+      const axeConfig = {
+        inject: true,
+        ...(browser.options.axe || {}),
+      };
       if (axeConfig.inject) {
         if (browser.execute('return window.axe === undefined;')) {
           if (!axeCoreSrc) {
             axeCoreSrc = fs.readFileSync(require.resolve('axe-core'), 'utf8');
             axeCoreSrc = axeCoreSrc.replace(/^\/\*.*\*\//, '');
+
+            const axeOptions = axeConfig.options;
+            if (axeOptions) {
+              axeCoreSrc = `${axeCoreSrc}\naxe.configure(${JSON.stringify(axeOptions)});`;
```

Comment:
> `${axeCoreSrc}\naxe.configure(${JSON.stringify(axeOptions)});`;

Is there a cleaner options than the `\n` between `${axeCoreSrc}` and `axe.configure(...` ?

Reply:
is needed to place the json config on a new line to not have syntax errors when loading the script.  maybe could break it apart and append each line separately, but I think this is ok as only appending 1 item to the end of the script

Reply:
we might be able to get away with a newline in the string literal

Reply:
One options for purely readability could be
```
const axeConfigrue = `axe.configure(${JSON.stringify(axeOptions)})`;
axeCoreSrc = `${axeCoreSrc}\n${axeConfigure};`;
```